pg_search/src/api/index.rs

```diff
@@ -34,7 +34,7 @@ use tantivy::schema::{FieldType, OwnedValue, Value};
 
 #[allow(clippy::type_complexity)]
 #[pg_extern]
-pub fn schema_bm25(
+pub fn schema(
     index_name: &str,
```

Comment:
I would prefer this function take a `PgRelation` instead of a string.  The reason is that Postgres will validate the user is providing a valid, and fully-qualified (or SEARCH_PATH included) relation name for us.

```suggestion
    index_name: PgRelation,
```

However, we'll also need to reopen the relation at the start of the function with an AccessShareLock:

```rust
let index_name = PgRelation::with_lock(index_name.oid(), pg_sys::AccessShareLock as _);
```

The reason for reopening it is to ensure a concurrent DROP/ALTER doesn't come in while we're reading it.